entrypoint.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -e
 
+# Copy config file if does not exists
+rsync -a -v --ignore-existing config.yaml.sample config/config.yaml
```

Comment:
Why are you doing this ?

Reply:
For the persistence of the mqtt config file. You want to create the file if it's missing in the config without overwriting it.

Reply:
I would not do that, I think it better to crash if the file doesn't exists instead of copying a bad config file. What do you think ?